src/ServiceControl.LoadTests.Reporter/ServiceControl.LoadTests.Reporter.csproj

```diff
@@ -1,17 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net462</TargetFramework>
-    <OutputType>Exe</OutputType>
-    <AssemblyName>QueueLengthMonitor</AssemblyName>
+    <TargetFramework>net461</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\ServiceControl.LoadTests.Messages\ServiceControl.LoadTests.Messages.csproj" />
+    <PackageReference Include="NServiceBus" Version="7.0.1" />
```

Comment:
ServiceControl references NSB. Transitive dependencies will resolve this so you can drop the package reference